gulpfile.js

```diff
@@ -73,7 +76,7 @@ gulp.task('test', ['common'], function() {
       fileName: 'client/perfkit_scripts.js',
       compilerFlags: {
         angular_pass: true,
-        compilation_level: 'SIMPLE_OPTIMIZATIONS',
+        compilation_level: 'WHITESPACE_ONLY',
```

Comment:
Was for debugging purposes.  Reverting to SIMPLE_OPTIMIZATIONS, since the new ES6 code being added won't work in WHITESPACE_ONLY mode.